From dcae873414ff643e1de790f256e414923e2aef8b Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 30 May 2013 11:14:39 +0200
Subject: [PATCH] systemd/emergency.service: do not run for
 action_on_fail=continue

same as for dracut-emergency.service
---
 modules.d/98systemd/emergency.service | 1 +
 1 file changed, 1 insertion(+)

diff --git a/modules.d/98systemd/emergency.service b/modules.d/98systemd/emergency.service
index 35d9c31..c19fe37 100644
--- a/modules.d/98systemd/emergency.service
+++ b/modules.d/98systemd/emergency.service
@@ -12,6 +12,7 @@ Description=Emergency Shell
 DefaultDependencies=no
 After=systemd-vconsole-setup.service
 Wants=systemd-vconsole-setup.service
+ConditionKernelCommandLine=!action_on_fail=continue
 
 [Service]
 Environment=HOME=/
